policies/golang/min_go_version.py

```
@@ -0,0 +1,41 @@
+            return tuple(int(p) for p in parts[:2])
+
+        try:
+            actual = parse_version(actual_version)
+            minimum = parse_version(min_version)
+
+            c.assert_true(
+                actual >= minimum,
+                f"Go version {actual_version} is below minimum {min_version}. "
+                f"Update go.mod to require Go {min_version} or higher."
+            )
+        except (ValueError, TypeError) as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)